packages/pds-ember/app/styles/pds/components/app/_config.scss

```diff
@@ -2,55 +2,60 @@ $_module: "PDS.Components.App";
 /* --- [debug] CONFIG: #{$_module} --- */
 @use "../../theme";
 @use "../../tokens/color";
-@use "../drawer/config" as Drawer;
 @use "../../utils/media" as Media;
+@use "../drawer/config" as Drawer;
 
-// Structure 1.8 (latest as of 2020-04-13)
-// expects a body copy of 14px (inherited from Bulma) -- (T_T)
-$fontSize: 14px; // TODO: inherit font-size from <body> (waiting on go-ahead from Design)
-$fontFamily: theme.$body-font-family;
-$backgroundColor: theme.$background-color;
+$fontSize: theme.$fontSize; // TODO: inherit font-size from <body>
+$fontFamily: theme.$fontFamily;
+$backgroundColor: theme.$backgroundColor;
 
 @mixin layout--pc {
   /* [debug] #{$_module}@layout--pc */
   /*
      auto       1fr
-    +---------+------+
-    | head    | head |
-    +---------+------+
-    | sidebar | main |
-    +---------+------+
-    | sidebar | foot |
-    +---------+------+
+    +---------+--------+
+    | head    | head   |
+    +---------+--------+
+    | banner  | banner |
+    +---------+--------+
+    | sidebar | main   |
+    +---------+--------+
+    | sidebar | foot   |
+    +---------+--------+
   */
   display: grid;
   grid-template-areas:
     "head    head"
+    "banner  banner"
```

Comment:
Nice, we'll be able to ditch our overrides here.

Reply:
that's the idea 😄